JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -121,6 +121,11 @@ public function normalize($object, $format = null, array $context = [])
             if ('id' === $attributeName || !$attributeMetadata->isReadable()) {
                 continue;
             }
+
+            if ($this->nameConverter) {
```

Comment:
It will not work at that line (https://github.com/jderusse/DunglasApiBundle/blob/fix-attributename-normalization/JsonLd/Serializer/ItemNormalizer.php#L129 will fail because the name is converted too early).

It should be done here IMO https://github.com/jderusse/DunglasApiBundle/blob/fix-attributename-normalization/JsonLd/Serializer/ItemNormalizer.php#L157